### PR TITLE
Fix S3 bucket storage creation

### PR DESCRIPTION
**Pull request by devopsdina**
- Remove whitespace
- Updating description for workflow inputs
- Remove reference to "dina" in workflow inputs
- Update Create S3 bucket logic
- Removing the default region to be "us-east-1", making the user input the region.  This avoids inattentive users not noticing the default region isn't the region they need/want.

### CODE REVIEW

**Comment by devopsdina** [comment]
@giofontana Please review, comment and approve/reject, but don't merge it!  I will merge it once we are good so I can set the squash commit message.

**Comment by giofontana** [comment]
Nice! Good enhancements! Already reviewed and for me, it's approved! :)

**Review by giofontana — APPROVED** [review]
Approved

**Pull request merged**
